implementations/rust/ockam/ockam_app/src/app/app_state.rs

```diff
@@ -1,7 +1,8 @@
-use std::sync::{Arc, RwLock};
+use std::sync::Arc;
 
-use miette::IntoDiagnostic;
-use tauri::async_runtime::{block_on, spawn};
+use miette::{miette, IntoDiagnostic};
+use tauri::async_runtime::{block_on, spawn, RwLock};
+use tracing::log::{error, info};
```

Comment:
```suggestion
use tracing::{error, info};
```
Rust Analyzer gets me on this one accidentally somewhat often.

Reply:
Done (I think I'm actually responsible for a fair amount of those. I'll be careful from now on)